app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -185,12 +185,12 @@ $(document).ready(function($) {
 });
 
 var get_total_cost = function() {
-  var num_rows = $('#payout_table tbody').find('tr').length;
+  var num_rows = $('#payout_table').find('tr').length;
   var total = 0;
   var i = 1;
 
   for (i = 1; i < num_rows; i += 1) {
```

Comment:
change to `for( let i = 0` ?

- Init `i` to 0 cause of [this](https://github.com/gitcoinco/web/pull/2386/files#r224689252) 
- declaring variable here as opposed to [L190](https://github.com/gitcoinco/web/pull/2386/files#r224689394)